recipe-server/client/control_new/components/common/EnvAlert.js

```diff
@@ -0,0 +1,66 @@
+import { Alert } from 'antd';
+import React from 'react';
+
+/**
+ * Simple component which tells the user whether they are viewing a dev or staging
+ * environment. On production, nothing is shown.
+ */
+export default class EnvAlert extends React.PureComponent {
+  static productionUrls = ['normandy-admin.prod.'];
+  static stageUrls = ['normandy-admin.stage.'];
+
+  static learnMoreLink = (
+    <a
+      href="http://normandy.readthedocs.io/en/latest/qa/environments.html"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Learn more about Normandy environments.
+    </a>);
+
+  /**
+   * Find a string fragment in an array of strings.
+   *
+   * @param  {String}        needle   String fragment to find.
+   * @param  {Array<String>} haystack String collection to search.
+   * @return {Boolean}                Was the fragment found?
+   */
+  static findPartialString(needle, haystack) {
+    return !!haystack.find(straw => straw.indexOf(needle) > -1);
+  }
+
+  static checkProduction(url) {
+    return EnvAlert.findPartialString(url, EnvAlert.productionUrls);
+  }
+
+  static checkStaging(url) {
+    return EnvAlert.findPartialString(url, EnvAlert.stageUrls);
+  }
+
+  render() {
+    const currentUrl = window.location.href;
```

Comment:
We can't actually, as `lit'l` only deals with 'internal' app routes. That is, the router doesn't know anything about the `location.origin`, the full URL, etc. The `router` state object only gives us app-relevant data (`you're on this Component on this route at the path /whatever/cool`).

Do you have any suggestions on stubbing out the `window` object? In the past, I've used something like [window-or-global](https://github.com/purposeindustries/window-or-global/blob/master/lib/index.js) for a similar case, but not sure if it would work here. I'm open to suggestions.